crates/ty_ide/src/selection_range.rs

```diff
@@ -66,20 +66,28 @@ x = 1 + <CURSOR>2
 
         assert_snapshot!(test.selection_range(), @r"
         info[selection-range]: Selection Range 0
+         --> main.py:1:1
+          |
+        1 | /
+        2 | | x = 1 + 2
+          | |__________^
+          |
+
+        info[selection-range]: Selection Range 1
```

Comment:
drive-by side-effect: the ModModule is always a valid Even Bigger selection range. I don't use this API so I have no sense of if this is good or bad.

Reply:
I think it's fine.